.github/workflows/deploy-integration-tests.yaml

```diff
@@ -57,6 +57,8 @@ jobs:
       deployment_id: ${{ secrets.PROVIDER_INTEGRATION_TESTS_DEPLOYMENT_ID }}
       astronomer_key_id: ${{ secrets.PROVIDER_INTEGRATION_TESTS_ASTRONOMER_KEY_ID }}
       astronomer_key_secret: ${{ secrets.PROVIDER_INTEGRATION_TESTS_ASTRONOMER_KEY_SECRET }}
+      organization_id: ${{ secrets.ORGANIZATION_ID }}
+      bearer_token: ${{ secrets.BEARER_TOKEN }
```

Comment:
We need to pass this for KE deployment too

Reply:
thanks, @pankajkoti added

Reply:
I though we could get it through this https://github.com/astronomer/astronomer-providers/blob/main/dev/integration_test_scripts/trigger_dag.py#L83?

Reply:
@Lee-W i have created a workspace token that will always be the same and we can use that as well instead of making a new call every time